examples/plot_pwdslope.py

```diff
@@ -8,29 +8,41 @@
 then used as a guide to smooth a noise realization following the structural
 dips of the input data via the :py:class:`pylops.signalprocessing.PWSmoother2D`.
 
-.. [1] Fomel, S., "Applications of plane‐wave destruction filters",
+.. [1] Fomel, S., "Applications of plane-wave destruction filters",
    Geophysics. 2002.
 
 """
 
+from functools import partial
+
 import matplotlib.pyplot as plt
 import numpy as np
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
 import pylops
+from pylops.signalprocessing import PWSmoother2D
 from pylops.utils.signalprocessing import pwd_slope_estimate, slope_estimate
 
 plt.close("all")
 np.random.seed(10)
 
+
+def create_colorbar(im, ax):
+    divider = make_axes_locatable(ax)
+    cax = divider.append_axes("right", size="5%", pad=0.1)
+    cb = fig.colorbar(im, cax=cax, orientation="vertical")
+    return cax, cb
+
+
 ###############################################################################
 # Sigmoid model
 # ------------------
-# To start we import the same `2d image <http://ahay.org/blog/2014/10/08/program-of-the-month-sfsigmoid/>`_
+# To start we import the same `2D image <http://ahay.org/blog/2014/10/08/program-of-the-month-sfsigmoid/>`_
 # that we used in the seislet transform example. This image contains curved
 # reflectors that we will try to follow during the smoothing operation.
 inputfile = "../testdata/sigmoid.npz"
 
-sigmoid = np.load(inputfile)["sigmoid"].T
+sigmoid = 1e3 * np.load(inputfile)["sigmoid"].T
```

Comment:
@cako any specific reason for adding 1e3 * ?

Reply:
Oh, I normally do this to sigmoid to bring it to order of magnitude ~1. Totally optional.